src/messages/message.rs

```diff
@@ -1175,3 +1198,85 @@ mod message_type_tests {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod jwe_flat_json_tests {
+    extern crate chacha20poly1305;
+    extern crate sodiumoxide;
+
+    use super::*;
+
+    use k256::elliptic_curve::rand_core::OsRng;
+    use utilities::{KeyPairSet, get_keypair_set};
+
+    #[test]
+    fn can_create_flat_jwe_jsons() -> Result<(), Error> {
```

Comment:
```suggestion
    fn can_create_flat_jwe_json() -> Result<(), Error> {
```

we only create a single json :)